docker-compose.yml

```
@@ -0,0 +1,23 @@
+version: '3'
+services:
+  postgres:
+    image: postgres
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=${POSTGRES_DB}
```

This is verbose. Following would be enough:
```suggestion
    environment:
      - POSTGRES_USER
      - POSTGRES_PASSWORD
      - POSTGRES_DB
```
Refer to [docker docs](https://docs.docker.com/compose/environment-variables/#pass-environment-variables-to-containers) for more info.